package.json

```diff
@@ -154,6 +154,7 @@
     "@types/node": "18.7.1",
     "@types/node-sass-tilde-importer": "1.0.0",
     "@types/sass": "1.43.1",
+    "@types/semver": "^7.3.12",
```

Comment:
We shouldn't use range characters in our package.json.

Reply:
Updated.